src/wallet/wallet.cpp

```diff
@@ -1156,6 +1156,11 @@ bool CWallet::AddToWallet(const CWalletTx& wtxIn, bool fFlushOnClose)
     if (!strCmd.empty())
     {
         boost::replace_all(strCmd, "%s", wtxIn.GetHash().GetHex());
+#ifdef WIN32
+        boost::replace_all(strCmd, "%w", GetName());
```

Comment:
This line is unsafe since it can corrupt wallet names and trigger arbitrary shell commands to run from an RPC. Suggested fix from previous thread https://github.com/bitcoin/bitcoin/pull/13339#discussion_r357655050 is 26e0017bcfca722892eaf79d459d80f57a599863